dotcom-rendering/src/web/browser/debug/debug.css

```diff
@@ -0,0 +1,43 @@
+/* Layout: Annotate */
+main[data-layout]::before {
+	content: 'layout: ' attr(data-layout);
```

Comment:
```suggestion
	content: '📰 layout: ' attr(data-layout);
```

Reply:
I wonder about the emojis and accessibility here, would this not read out as "Newspaper layout..." which is quite confusing.

Reply:
I'd be tempted to go for `Page layout` here, or something like that? Just because we also have things like `ContainerLayout`, which are different

Reply:
I've gone with page layout - mostly for descriptiveness, though the emojis are kind of cool!